### ISSUE TITLE
1.4.1 and 1.4.3 for focused controls in Android

### ISSUE THREAD

**Issue by mitchellevan** [w3c/matf#112]
Android's default focus indicator is sometimes described as a gray background. I would describe it as a lightness change, which applies to both the foreground and background of a control.

I accept that Android's default focus indicator is "determined by the user agent", so SC 1.4.11 Non-text Contrast does not apply to it.

My question: Does the focused state of an Android native control have similar exemptions from [SC 1.4.1 Use of Color](https://www.w3.org/TR/WCAG2/#use-of-color) and [SC 1.4.3 Contrast (Minimum)](https://www.w3.org/TR/WCAG2/#contrast-minimum)?

Here is an argument in favor of exemption. [Native or Custom? A Guide to Recognizing Focus Styles](https://www.tpgi.com/native-or-custom-a-guide-to-recognizing-focus-styles/)

<blockquote>
An important remark for mobile focus indication on Android is that, in most cases, the native focus indicator is a semi-opaque overlaid color tint. It is a subtle change that can be difficult to locate on screen. Given that it is just a color change, does it fail 1.4.1 Use of Color? And what if the color overlay causes text underneath to have lower contrast, does that fail 1.4.3 Contrast (Minimum)? This is what it states in <a href="https://www.w3.org/TR/WCAG22/#dfn-contrast-ratio">WCAG’s notes about contrast</a>.

> NOTE 6
> WCAG conformance should be evaluated for color pairs specified in the content that an author would expect to appear adjacent in typical presentation. Authors need not consider unusual presentations, such as color changes made by the user agent, except where caused by authors’ code.

This means that 1.4.1 Use of Color and 1.4.3 Contrast (Minimum) issues generated by the use of such focus indicators are exempted of those criteria. In other words, no failure to report.
</blockquote>
